base_import_default_enable_tracking/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================
+Base Import Default Enable Tracking
+===================================
+
+This modules simply enables history tracking when doing an import.
```

Comment:
This feature still exists but it's only available in debug mode. So it's not possible for a simple user to enable tracking before doing an import. If you install this module, tracking will always be enabled, and only a user who is in debug mode will be able to disable it.